Updated MSI sample to use access token to get a secret from a key vault (#6944)
AzureAD/microsoft-authentication-library-for-js: samples/msal-node-samples/managed-identity/README.md

# Managed Identity for Azure VM Sample

> This sample demonstrates how to use [managed identity via the msal-node library](/lib/msal-node/docs/managed-identity.md) to retrieve tokens for a managed identity application running on an Azure VM, and then use the token to retrieve a secret from a key vault.

Reviewer: ```suggestion
This sample demonstrates how to use [managed identity via the msal-node library](/lib/msal-node/docs/managed-identity.md) to acquire a token for accessing Azure KeyVault, on behalf of a managed identity configured on an Azure VM. The sample then calls the downstream api - it calls KeyVault and retrieves a secret.
```

};
```

> Additionally, the `KEY_VAULT_NAME` and `SECRET_NAME` values need to be updated.

Reviewer 2: ```suggestion
Additionally, update the `KEY_VAULT_NAME` and `SECRET_NAME` in index.ts file.
```
